Source/Render/sokol/CMakeLists.txt

```diff
@@ -20,7 +20,7 @@ IF (OPTION_PROCESS_SHADERS)
 
     if(NOT DEFINED ENV{PERIMETER_SHADER_LANG})
         #Use all GLSL dialects, HLSL for D3d and WebGPU
-        set(PERIMETER_SHADER_LANG "glsl410:glsl300es:hlsl5:wgsl")
```

Comment:
`wsgl` does not support `sampler2DShadow` but in Sokol examples they use Open GL ES for this:

```
if (FIPS_EMSCRIPTEN)
    if (FIPS_EMSCRIPTEN_USE_WEBGPU)
        set(sokol_backend SOKOL_WGPU)
        set(slang "wgsl")
    else()
        set(sokol_backend SOKOL_GLES3)
        set(slang "glsl300es")
    endif()
```

Reply:
Interesting, is fine since we dont use wgpu but good to remember in future in case is needed back